### PR TITLE
Coating protections

### PR DESCRIPTION
**Pull request by slacAWallace**
Basic coating protection per @mseaberg  request.

### CODE REVIEW

File: lcls-plc-lfe-optics/lfe_optics/POUs/P_CoatingProtections.TcPOU [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            sLowerCoatingType := 'B4C/Ni');
+
+    	fbStripProtMR2L0 : FB_MirrorTwoCoatingProtection := (
+            sDevName := 'MR2L0',
+            nUpperCoatingBoundary := 12800000,
+            nUpperCoatingBitmask := 2#1111_1100_0000_0000,
+            sUpperCoatingType := 'Ni',
+            nLowerCoatingBoundary := 15998960,
+            nLowerCoatingBitMask := 2#0000_0011_1111_1111,
+            sLowerCoatingType := 'B4C');
```
Comment (mseaberg) [inline lcls-plc-lfe-optics/lfe_optics/POUs/P_CoatingProtections.TcPOU:22]: It looks like Ni is off by one, I think we want 1111_1110_0000_0000

Comment (slacAWallace) [reply lcls-plc-lfe-optics/lfe_optics/POUs/P_CoatingProtections.TcPOU]: Should be fixed now.

File: lcls-plc-lfe-optics/lfe_optics/POUs/P_CoatingProtections.TcPOU [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            sLowerCoatingType := 'B4C/Ni');
+
+    	fbStripProtMR2L0 : FB_MirrorTwoCoatingProtection := (
+            sDevName := 'MR2L0',
+            nUpperCoatingBoundary := 12800000,
+            nUpperCoatingBitmask := 2#1111_1110_0000_0000,
+            sUpperCoatingType := 'Ni',
+            nLowerCoatingBoundary := 15998960,
+            nLowerCoatingBitMask := 2#0000_0011_1111_1111,
+            sLowerCoatingType := 'B4C');
```
Comment (mseaberg) [inline lcls-plc-lfe-optics/lfe_optics/POUs/P_CoatingProtections.TcPOU:22]: Coating boundaries look off to me. I think it should be
```suggestion
    	fbStripProtMR2L0 : FB_MirrorTwoCoatingProtection := (
            sDevName := 'MR2L0',
            nUpperCoatingBoundary := 14300000,
            nUpperCoatingBitmask := 2#1111_1110_0000_0000,
            sUpperCoatingType := 'Ni',
            nLowerCoatingBoundary := 12800000,
            nLowerCoatingBitMask := 2#0000_0011_1111_1111,
            sLowerCoatingType := 'B4C');
```

Comment (slacAWallace) [reply lcls-plc-lfe-optics/lfe_optics/POUs/P_CoatingProtections.TcPOU]: Yes, definitely a mistake. Should be corrected now.

**Review by mseaberg — APPROVED** [review]

**Pull request merged**